core/src/file/qt_fs_browser.cc

```diff
@@ -16,8 +16,9 @@
 ** For more information : contact@centreon.com
 */
 
-#include <QDir>
-#include <QFile>
+#include <sys/types.h>
+#include <regex.h>
```

Comment:
Should not we remove this?

Reply:
not yet regcomp is used in qt_fs_browser.